### ISSUE TITLE
Mail notifications

### ISSUE THREAD

**Issue by Dark-Fenix** [borgbackup/borg#1626]
Hi,

Is there any built-in method for borgbackup to send mails when a job is done (if planned in a cron) ? And to limit mail alerts to failed jobs only (with logs and errors) ?

If not, will it be part of this wonderful project ?

Thanks ! :)

**Comment by RonnyPfannschmidt** [comment]
im not a maintainer, but i think that is outside of scope (in particular sice surrounding scripts can be trivial)

also there are very intense differences in oppinion on how/when to do this, how to do authentication and many other topics

as such i think its best to script this around borg

**Comment by Ranlvor** [comment]
If you have a cronjob cron itself already emails you when the job did print anything. And borg should print something if there are problems, so your setup already has the feature you request.

**Comment by Dark-Fenix** [comment]
Hi,

Tanks for your replies.

I have set a simple bash script to call borgbackup (on Debian Jessie), but I don't have any mail from my cron when a run fails (or even if it is ok). Maybe I have a misconfiguration, but I deployed Borg over about a dozen machines... The Borg output is ok when manually launched, but when I put it in my script (cron-planned), I can't handle anything from stdout/stderr...

So nervermind, this is my concern. But even if I agree this could be outside of scope, as there is already a web-interface under development I really think it could be nice for lambda-users to simply specify a mail address to have a beautiful report every morning :)
